src/main/java/stepsdesign/ControlCenterSteps.java

```diff
@@ -0,0 +1,205 @@
+  public void clickCloseTabDatasetEntity() {
+    CdfControlCenterActions.clickOnCloseTabDatasetEntity(); }
+
+  @Then("Select dropdown : {string} with option value: {string} in control center")
+  public void selectDropdownFilterOptionValue(String pluginProperty, String option) {
+    CdfControlCenterActions.selectFilterDropdownOption(pluginProperty, option); }
+
+  @Then("Verify user is able to see all entities displayed message on the basis of filter selection")
+  public void displayFilterDatasetApplicationArtifactsMessage() {
+    CdfControlCenterActions.clickOnDatasetApplicationArtifactsMessage(); }
+
+  @Then("Click on the schema link of the dataset entity details page")
+  public void clickOnSchemaTabLink() {
+    CdfControlCenterActions.clickOnSchemaTabDataset(); }
+
+  @Then("Verify user is navigated to the schema details page of the dataset entity page")
+  public void navigateToSchemaTabLinkPage() {
+    CdfControlCenterActions.navigateToSchemaTabPage(); }
+
+  @Then("Click on the programs link of the dataset entity details page")
+  public void clickOnProgramsTabLink() {
+    CdfControlCenterActions.clickOnProgramsTabDataset(); }
+
+  @Then("Verify user is navigated to the programs details page of the dataset entity page")
+  public void navigateToProgramsTabLinkPage() {
+    CdfControlCenterActions.navigateToProgramsTabPage(); }
+
+  @Then("Select the sort by dropdown with option value: {string}")
+  public void selectDropdownSortOptionValue(String option) {
+    CdfControlCenterActions.selectSortDropdown();
+    CdfControlCenterActions.selectSortDropdownOptionValue(option); }
+
+  @Then("Verify user is able to see the entities sorted by the newest option")
+  public void clickOnNewestOptionMessage() {
+    CdfControlCenterActions.clickOnNewestOptionMessage(); }
+
+  @Then("Verify user is able to see the entities sorted by the oldest option")
+  public void clickOnOldestOptionMessage() {
+    CdfControlCenterActions.clickOnOldestOptionMessage(); }
+
+  @Then("Verify user is able to see the entities sorted by the Z to A option")
+  public void clickOnZToAOptionMessage() {
+    CdfControlCenterActions.clickOnZToAOptionMessage(); }
+
+  @Then("Verify user is able to see the entities sorted by the A to Z option")
+  public void clickOnAToZOptionMessage() {
+    CdfControlCenterActions.clickOnAToZOptionMessage(); }
+
+  @Then("Click on the plus add button of the dataset entity to add the tags successfully")
+  public void clickOnAddTagIcon() {
+    CdfControlCenterActions.clickOnAddTagIcon(); }
+
+  @Then("Verify user is able to enter the values in tag input field: {string}")
+  public void enterValueInTagInputField(String value) {
+    CdfControlCenterActions.enterValueTagInputField(value);
+    CdfControlCenterActions.pressEnterKey();
+  }
+
+  @Then("Verify that the tag count of dataset entity increases when the user adds the tag")
```

Comment:
Verify the tag count of dataset entity when user adds the tag

Reply:
changed